src/basic_stack.FOR

```diff
@@ -0,0 +1,111 @@
+      PROGRAM TEST1
+C
+C  THIS PROGRAM CONTAINS SEVERAL FUNCTIONS THAT WILL
+C  ALLOW TESTING OF FORTRAN STACK UNWINDING
+C  OPTIONS COMPILED WITH:
+C  -g -O0 -Wall -std=legacy
+C
+#if NOINLINE
+      !GCC$ ATTRIBUTES NOINLINE :: A1, A2, B1, B2, C1, C2, TOP1, TOP2
+#endif
+C  DECLARE FUNCTIONS
+C
+      INTEGER A1, A2
+      INTEGER RES
+C
```

Comment:
Just curious, are these comments some Fortran convention?

Reply:
Yes, that's the format of Fortran comments - the first field is the letter C.

This works because the compiler only looks at column 7 and later for valid Fortran code, except for PreProcessor directives, which the compiler will never see anyway.